pczt/tests/end_to_end.rs

```diff
@@ -495,8 +496,8 @@ fn zsa_to_zsa() {
         note
     };
 
-    let isk = IssuanceAuthorizingKey::from_bytes([1; 32]).unwrap();
-    let ik = IssuanceValidatingKey::from(&isk);
+    let isk = IssueAuthKey::from_bytes(&[1u8; 32]).unwrap();
```

Comment:
I prefer to specify that it is a ZSASchnorr validating key to avoid any ambiguity.
 ```suggestion
    let isk = IssueAuthKey::<ZSASchnorr>::from_bytes(&[1u8; 32]).unwrap();
```